src/main/java/com/tozny/sdk/realm/methods/user_push/UserPushRequest.java

```diff
@@ -0,0 +1,54 @@
+package com.tozny.sdk.realm.methods.user_push;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.tozny.sdk.ToznyApiRequest;
+
+import javax.annotation.Nullable;
+
+/**
+ * Collects request parameters for invoking the "realm.user_push" method.
+ * Requires that either the `user_id` or `email` or `username` field be set.
+ */
+@JsonAutoDetect(getterVisibility=JsonAutoDetect.Visibility.NONE)
+public class UserPushRequest implements ToznyApiRequest {
+
+    @JsonProperty private String method = "realm.user_push";
+    @JsonProperty private String session_id;
+    @JsonProperty private String user_id;
+    @JsonProperty private String email;
+    @JsonProperty private String username;
+
+    public UserPushRequest(String session_id, @Nullable String user_id, @Nullable String email, @Nullable String username) {
```

Comment:
Everything is built with the same pattern for some fancy dispatch/serialization work. While I agree in principal, refactoring this one call to follow a different pattern is more change than we want to introduce for just this one quick API call addition. We do have a rewrite of the SDK targeted for a future date already that will help clarify a lot of this, though, and that will be a good change for down the road.